play-json/src/main/scala/play/api/libs/json/JsonConfiguration.scala

```diff
@@ -3,7 +3,10 @@
  */
 package play.api.libs.json
 
-case class JsonConfiguration(naming: JsonNaming)
+case class JsonConfiguration(
+  naming: JsonNaming,
```

Comment:
Can we define all the default values here, e.g. `naming: JsonNaming = JsonNaming.Identity`. That also means we can change the `defaultConfiguration` to just `JsonConfiguration()`.

This is nice when you want to customize the configuration from the default, since all you need to do is `JsonConfiguration(useDefaultValues = true)`